src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const stringSplitted = sourceString.split(';');
```

Comment:
[CODE STYLE] - The variable name `stringSplitted` is grammatically incorrect and it doesn't provide clear information about what this variable is used for. Consider renaming it to `splitStyles` or `styleItems`.